### PR TITLE
fix(DB/Gameobject): Add (more) Tailasher Eggs

### LINKED ISSUE #5343: [Quest] Break a Few Eggs
## CURRENT BEHAVIOUR:
Quest: https://wowgaming.altervista.org/aowow/?quest=815
Tailasher Egg: https://wowgaming.altervista.org/aowow/?item=4890
Currently these are the locations of the tailasher eggs:

- [x] 68.2 88.6
- [x] 68.3 71.9
- [x] 67.1 71.4
- [x] 61.4 78.3
- [x] 62.1 96.1
- [x] 64.5 73.3
- [x] 64.9 82.4
- [x] 69.2 82.2
- [x] 69.7 70.3
- [x] 69.7 74.7
- [x] 59.8 89.6
- [x] 60.3 82.8
- [x] 59.4 83.7

## EXPECTED BLIZZLIKE BEHAVIOUR:

All the locations above and more should exist. These are the ones that are missing:
- [ ] 60.9 78.8
- [ ] 63.1 94.5
- [ ] 63.5 74.4
- [ ] 63.9 83.7
- [ ] 63.9 86.8
- [ ] 66.8 89.0
- [ ] 67.2 80.6
- [ ] 67.8 69.9
- [ ] 68.0 80.4
- [ ] 68.4 86.9
- [ ] 68.7 74.4
- [ ] 68.9 71.1
- [ ] 70.2 71.0
- [ ] 60.0 88.0

All of the eggs should be present, making it more consistent.

## SOURCE:

https://classic.wowhead.com/quest=815/break-a-few-eggs
Since I am using classic wowhead as my source, I had to go through every cordinate and compare it with the comments.

Some of the coordinates that are on wowhead had to be excluded since:
1. They were not consistent compared to the comments (eggs should be guarded by raptors).
2. They would be too close to other eggs.
3. The coordinates were not precise enough.

## STEPS TO REPRODUCE THE PROBLEM:

1. Use the command `.go zonexy 59.4 83.7 14`.
2. Compare to location of tailasher eggs according to wowhead.
[Optional] 3. Go to a coordinate that is correct. i.e  `.go zonexy  68.2 88.6 14` and compare with wowhead again.

## EXTRA NOTES:

The coordinates from wowhead are not perfect, which means the eggs have to be added manually and reviewed.

I intend to resolve this myself, I will await a external tester to confirm this aswell.

While playing on chromiecraft I noticed the 'Opening' spell cast was incorrect but I could not reproduce this on my local test server, might be caused by the fact Chromiecraft is a few commits behind.

## AC HASH/COMMIT:

3a5d6fd001bb066e582da4034cb7e7d7266c662b

## OPERATING SYSTEM:

Windows 10

## MODULES:

N/A

## CUSTOMIZATIONS:

N/A

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/97909895-quest-break-a-few-eggs?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Branel**
## Changes Proposed:
-  Add new spawn location for Tailasher Egg's (3240 object id)

## Issues Addressed:
- Closes #5343

## SOURCE:

https://classic.wowhead.com/quest=815/break-a-few-eggs

## Tests Performed:

- Tested on Windows 10 on my local game server.

## How to Test the Changes:

.go zonexy 68.2 88.6 14 to see one of the new spawn locations, please see the original bugtracker for more coordinates to test, or compare with wowheads map of locations for the tailasher eggs. Beware that some of the locations on wowhead are inaccurate and I have decided to not include them.

## Known Issues and TODO List:

N/A

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1618745790841763500.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+INSERT INTO `gameobject` VALUES (2135378, 3240, 1, 0, 0, 1, 1, -1142.09, -5103.27, 4.41125, 4.54915, -0, -0, -0.762402, 0.647104, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135379, 3240, 1, 0, 0, 1, 1, -969.367, -5182.59, 1.80058, 1.82394, -0, -0, -0.790712, -0.612188, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135385, 3240, 1, 0, 0, 1, 1, -813.653, -5316.14, 3.31857, 4.71632, -0, -0, -0.705714, 0.708497, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135387, 3240, 1, 0, 0, 1, 1, -1251.37, -5341.21, 5.36343, 4.04087, -0, -0, -0.900604, 0.43464, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135386, 3240, 1, 0, 0, 1, 1, -1140.34, -5337.26, 5.67458, 4.04087, -0, -0, -0.900604, 0.43464, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135384, 3240, 1, 0, 0, 1, 1, -1518.92, -5304.11, 7.73214, 3.61284, -0, -0, -0.972369, 0.233449, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135388, 3240, 1, 0, 0, 1, 1, -1327.5, -5496.98, 5.54005, 2.56263, -0, -0, -0.958391, -0.285457, 900, 0, 1, '', 0);
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1618745790841763500.sql:16]: hello! thanks for your contribution @Branel 

just some minor notes:

- you should always use `DELETE` before `INSERT`
- you should combine multiple `INSERT` statements into one

for more details, please read: https://www.azerothcore.org/wiki/Dealing-with-SQL-files

Comment (Branel) [reply data/sql/updates/pending_db_world/rev_1618745790841763500.sql:16]: Thank you for the feedback!

File: data/sql/updates/pending_db_world/rev_1618745790841763500.sql [hunk 2]
```diff
@@ -0,0 +1,18 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1618745790841763500');
+INSERT INTO `gameobject` VALUES
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1618745790841763500.sql:2]: missing `DELETE` before `INSERT` :

```sql
DELETE FROM `gameobject` WHERE guid IN (2135397, 2135398, 2135389, 2135391, 2135392, 2135393, 2135395, 2135396, 2135378, 2135379, 2135385, 2135387, 2135386, 2135384, 2135388);
```

**Comment by pklloveyou** [comment]
Tested

**Comment by pklloveyou** [comment]
Remember to add delete

**Comment by Branel** [comment]
> Tested

Thank you for testing.

Sorry for missing the delete.

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
approved, good job @Branel thanks @pklloveyou for checking this

**Pull request merged**